### DISCUSSION TITLE
new public hosting to replace the defunct service from podsync.net

### DISCUSSION THREAD

**Discussion by allanlaal** [mxpv/podsync#261] [General]
anyone up to host this?

@mxpv thank you for hosting Podsync for us in those years :)

"Fixes": #100 #37 #34

**Comment by allanlaal** [comment]
@mxpv can you share load and bw data per 100/1000 users so the potential hosters can estimate what they are getting in to? :)

**Comment by allanlaal** [comment]
per #38 it seems this replacement can only be a paid package to pay for the YouTube API requests

**Comment by allanlaal** [comment]
@mxpv before #38 did you consider creating a service where everyone can just add their own YouTube API key or use a very very restricted public version?

**Comment by jtagcat** [comment]
I can host stuff.
